### ISSUE TITLE
Set webhook payload items to something more relevant for cancelled builds

### ISSUE THREAD

**Issue by gerbil** [tcplugins/tcWebHooks#56]
Can't see webhooks for cancelled jobs

**Comment by netwolfuk** [comment]
I'm not sure what you are expecting to see. Can you please give more detail?

**Comment by gerbil** [comment]
I'm expecting to see hook for cancelled job. But i can't see any. 
Result for this job is stil running:
"resultPrevious": failure
"result": running

**Comment by netwolfuk** [comment]
Ah, I see what you mean. Although the `buildStatus` and `notifyType` do clarify that the build was cancelled (interrupted).

```
    "buildStatus": "Canceled (Tests passed: 41; exit code 137 (new))",
    "buildResult": "running",
    "buildResultPrevious": "failure",
    "buildResultDelta": "unknown",
    "notifyType": "buildInterrupted",
```

**Comment by netwolfuk** [comment]
`buildResult` needs to be set to something and `buildResultDelta` calculated.

**Comment by netwolfuk** [comment]
This is available for testing in the [v1.2.0-alpha.3](https://github.com/tcplugins/tcWebHooks/releases/tag/v1.2.0-alpha.3) release.
